source/pages/Tokens/CustomToken.tsx

```diff
@@ -24,6 +34,16 @@ export const CustomToken = () => {
     errorType: '',
     message: '',
   });
+  const [tokenMetadataInfos, setTokenMetadataInfos] =
+    useState<IAddCustomTokenMetadataInfos | null>(null);
+  const [tokenDecimalsWarning, setTokenDecimalsWarning] = useState({
```

Comment:
we can create at the top of the file something like an initial value and use it in these two states

```ts
const TOKENS_WARNING_INITIAL_VALUE = {
error: false,
value: '',
}
.
.
.
const [tokenDecimalsWarning, setTokenDecimalsWarning] = useState(TOKENS_WARNING_INITIAL_VALUE)
const [tokenSymbolWarning, setTokenSymbolWarning] = useState(TOKENS_WARNING_INITIAL_VALUE)
```